python/test_dcr_template.py

```diff
@@ -358,35 +382,6 @@ def test_persist_dcr_model_roundtrip(self):
         for key in param_ref.keys():
```

Comment:
I've expanded the test to check the values of int, float, array, and string values, though not arbitrary complex data structures such as wcs or bandpass.